src/inspector_agent.cc

```diff
@@ -215,7 +215,7 @@ class JsBindingsSessionDelegate : public InspectorSessionDelegate {
     Local<Value> argument = v8string.ToLocalChecked().As<Value>();
     Local<Function> callback = callback_.Get(isolate);
     Local<Object> receiver = receiver_.Get(isolate);
-    callback->Call(env_->context(), receiver, 1, &argument);
+    static_cast<void>(callback->Call(env_->context(), receiver, 1, &argument));
```

Comment:
No comment on the portability of the pragma, but there is no reason to cast to void _other_ than to avoid warnings and visually document that the return value is explicitly being ignored, I don't think a comment is needed for that usage.